semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/nodemethods/MethodMethods.scala

```diff
@@ -7,17 +7,17 @@ import io.shiftleft.Implicits.JavaIteratorDeco
 
 class MethodMethods(val node: nodes.Method) extends AnyVal {
 
-  def parameter: NodeSteps[nodes.MethodParameterIn] =
-    node.start.parameter
+  def parameter: Iterator[nodes.MethodParameterIn] =
+    node._methodParameterInViaAstOut
```

Comment:
I know the type is called `MethodParameterIn`, but we usually just refer to it as `parameter` throughout the rest of the code. Or is this generated?

Reply:
`_methodParameterInViaAstOut` is generated